packages/shared/src/utils/helpers/market-underlying.ts

```diff
@@ -33,7 +34,10 @@ export const getMarketInformation = (shortcode: string) => {
 export const getMarketName = (underlying: string) =>
     underlying ? getMarketNamesMap()[underlying.toUpperCase() as keyof typeof getMarketNamesMap] : null;
 
-export const getTradeTypeName = (category: string) =>
-    category
-        ? (getContractConfig()[category.toUpperCase() as keyof typeof getContractConfig] as TTradeConfig).name
-        : null;
+export const getTradeTypeName = (category: string, is_high_low = false, show_button_name = false) => {
+    const trade_type =
+        category &&
+        (getContractConfig(is_high_low)[category.toUpperCase() as keyof typeof getContractConfig] as TTradeConfig);
+    if (!trade_type) return null;
+    return (show_button_name && trade_type.button_name) || trade_type.name || null;
+};
```

Comment:
this function is doing the same as before but has more params to process (copied from Reports)